app/facades/movies_facade.rb

```diff
@@ -11,4 +16,14 @@ def top_20_rated
       Movie.new(movie_data)
     end
   end
+
+  def keyword_results
+    service = MoviesService.new
+
+    json = service.keyword_search(@keywords)
+
+    json[:results].map do |movie_data|
+      Movie.new(movie_data)
+    end
+  end
```

Comment:
because these two methods are almost identical, I'm wondering if we can refactor this into a single method in the future